src/overlays/actors/ovl_Bg_Dblue_Balance/z_bg_dblue_balance.h

```diff
@@ -7,7 +7,7 @@ struct BgDblueBalance;
 
 typedef void (*BgDblueBalanceActionFunc)(struct BgDblueBalance*, PlayState*);
 
-#define BGDBLUEBALANCE_GET_7F(thisx) ((thisx)->params & 0x7F)
+#define BGDBLUEBALANCE_GET_SWITCH_FLAG(thisx) ((thisx)->params & 0x7F)
```

Comment:
Any particular reason to split switch and flag with an underscore? Kinda feel like it just makes the macro longer and it's already a long macro, or is it weird for macros to have merged words?

Reply:
I find it a bit easier to read with the underscore since it's a long word, and it's a little more common to split two words with underscore. That said, shorter words like `CueId` are done as 1 word: `CUEID`, but I like it as 1 just because it's a shorter word. This all comes down to personal preference though, so I can keep `SWITCHFLAG` as 1 word if that's what we prefer.